pkgs/top-level/all-packages.nix

```diff
@@ -6959,7 +6959,7 @@ with pkgs;
   halibut = callPackage ../tools/typesetting/halibut { };
 
   halide = callPackage ../development/compilers/halide {
-    llvmPackages = llvmPackages_9;
+    llvmPackages = llvmPackages_12;
```

Comment:
Halide 14 should really be built against LLVM 14, if you can manage it. LLVM 12 is actually older than Halide 14 supports. The rule is that Halide version N claims compatibility with LLVM N, N-1, and N+1

Reply:
Oh thx for the review and for teaching me that! Gimme me a few minutes.

Reply:
@alexreinking since you seem knowledgeable about Halide, could you please test the package to some extent? I tested only compiling one tutorial lesson (I'm starting to learn it).

Reply:
I just moved into a new house day before yesterday, so now is not a great time. The earliest I could look at it is next Monday, July 11th.

Reply:
It compiled with llvm 14, thx :D

Reply:
> > since you seem knowledgeable about Halide
> 
> I am a maintainer of Halide and wrote my Master's thesis on it :)
> 
Oh niceee :D, now that looked into your profile picture more calmly, I've watched you presentation on CppCon! Very well explained, I wish I can learn Halide for my research. Such a nice surprise to see you're involved with Nix.
> > could you please test the package to some extent?
> 
> I just moved into a new house day before yesterday, so now is not a great time. The earliest I could look at it is next Monday, July 11th.
Do not worry, whenever you are available. Let's trust my testing for now haha.